Week2/wadeea.js

```diff
@@ -0,0 +1,246 @@
+// 3.4 If I initialize the variable x here, I will get a syntax error,
+// because It's already been declared. So I've only defined it.
+x = 7;
+// 3.5
+console.log(`The value of x will be: 7`);
+// 3.6
+console.log(x);
+
+// 4.
+let y = `This is a string`;
+// 4.1
+console.log(`The value of the string will be the following sentence: This is a string`);
+// 4.2
+console.log(y);
+// 4.3
+y = `New string`;
+// 4.4
+console.log(`The value will be the following: New string`);
+// 4.5
+console.log(y);
+
+// 5. The nearest integer is 7 because it's less than 7.50
+// 5.1
+let z = 7.25;
+// 5.2
+console.log(z);
+// 5.3
+let a = 7;
+// 5.4
+console.log(a);
+// 5.5
+let theHighestValue;
+if (a > z) {
+  theHighestValue = a;
+} else {
+  theHighestValue = z;
+}
+// 5.6
+console.log(`The highest value is: ` + theHighestValue);
+
+// 6.1
+const citiesInNetherlands = [];
+// 6.2
+console.log(`The value of the array is: 0`);
+// 6.3
+console.log(citiesInNetherlands);
+// 6.4
+const favoriteAnimals = [`cat`, `dog`, `hamster`, `lion`, `tiger`];
+// 6.5
+console.log(favoriteAnimals);
+// 6.6
+favoriteAnimals.push(`baby pig`);
+// 6.7
+console.log(favoriteAnimals);
+
+// 7.1
+const myString = `this is a test`;
+console.log(myString);
+// 7.2
+myString.length;
+// 7.3
+console.log(myString.length);
+
+// 8.1
+const myFirstName = `Wadeea`;
+const myLastName = `Kiwan`;
+let myAge = 33;
+const myTelephone = 0612345678;
+const genderMale = true;
+const childrenFirstNames = [`Kady`, `Aram`];
+// 8.2
+console.log(myFirstName);
+console.log(myLastName);
+console.log(myAge);
+console.log(myTelephone);
+console.log(genderMale);
+console.log(childrenFirstNames);
+// 8.3
+console.log(`The type of the variable 'myFirstName' will be: string`);
+console.log(`The type of the variable 'myLastName' will be: string`);
+console.log(`The type of the variable 'myAge' will be: number`);
+console.log(`The type of the variable 'myTelephone' will be: number`);
+console.log(`The type of the variable 'genderMale' will be: boolean`);
+console.log(`The type of the variable 'childrenFirstNames' will be: object`);
+// 8.4
+console.log(typeof myFirstName);
+console.log(typeof myLastName);
+console.log(typeof myAge);
+console.log(typeof myTelephone);
+console.log(typeof genderMale);
+console.log(typeof childrenFirstNames);
+// 8.5
+// 8.6
+typeof myFirstName === typeof myLastName;
+typeof myFirstName === typeof myAge;
+typeof myFirstName === typeof myTelephone;
+typeof myFirstName === typeof genderMale;
+typeof myFirstName === typeof childrenFirstNames;
+typeof myLastName === typeof myAge;
+typeof myLastName === typeof myTelephone;
+typeof myLastName === typeof genderMale;
+typeof myLastName === typeof childrenFirstNames;
+typeof myAge === typeof myTelephone;
+typeof myAge === typeof genderMale;
+typeof myAge === typeof childrenFirstNames;
+typeof myTelephone === typeof genderMale;
+typeof myTelephone === typeof childrenFirstNames;
+typeof genderMale === typeof childrenFirstNames;
```

Comment:
Again, these are simply expressions which don't really add much value to our program as we don't use them anywhere. It would be better if you maybe printed these values as `console.log(typeof myFirstName === typeof myLastName)`. I liked how you followed the naming conventions while creating your variables. Thumbs up for that.

Reply:
Thanks a lot 🙂. I'll print them.